src/services/socketService.ts

```diff
@@ -15,3 +15,5 @@ const createSocketService = () => {
 };
 
 export const socketService = createSocketService();
+
+export type SocketService = typeof socketService;
```

Comment:
nit: I did tell you do this, but it isn't something I'd do if I was writing this as a TS file from scratch. As discussed in the presentation, I'd make the type first: What should the service _do_.
```ts
type SocketService = {
  get(): SocketIOClient.Socket;
};

const createSocketService: (): SocketService => {
  ...
```